qtglean/pings.yaml

```diff
@@ -0,0 +1,22 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# This file defines the pings that are recorded by the Glean SDK.
+# Their code APIs is automatically generated, at build time using,
+# the `glean_parser` PyPI package.
+
+---
+$schema: moz://mozilla.org/schemas/glean/pings/2-0-0
+
+vpnsession:
+  description: |
+    Data for one VPN session (start/end time, session-based metrics, etc.)
+  include_client_id: true
+  send_if_empty: true
```

Comment:
Why?

Reply:
This ping is going to be imperfect - it doesn't track a user turning the VPN on/off from mobile system settings (that is coming separately), and so there are situations where it may record 0 or 2+ start datestamps, when ideally it records just one. I initially set this as `true` as a safety valve - so that we can send it and try to unwind these sticky situations, if we wanted to.

In reality, the code is going to submit this right after we record an ending datestamp, and so it would effectively never actually be empty. And so I've updated this to `false`, the default.